src/components/FormTextArea.vue

```diff
@@ -17,86 +17,133 @@
     </div>
     <textarea
       v-else
-      v-bind="$attrs"
-      :rows="rows"
-      :readonly="readonly"
       v-uni-id="label"
+      v-bind="objectOfAttrs"
       class="form-control"
+      :rows="rows"
+      :readonly="readonly"
       :class="classList"
       :name="name"
       :value="value"
       @input="$emit('input', $event.target.value)"
     />
-    <display-errors v-if="error || (validator && validator.errorCount)" :name="name" :error="error" :validator="validator"/>
-    <small v-if='helper' class='form-text text-muted'>{{helper}}</small>
+    <display-errors
+      v-if="error || (validator && validator.errorCount)"
+      :name="name"
+      :error="error"
+      :validator="validator"
+    />
+    <small v-if="helper" class="form-text text-muted">{{ helper }}</small>
   </div>
 </template>
 
 <script>
-import { createUniqIdsMixin } from 'vue-uniq-ids'
-import ValidationMixin from './mixins/validation'
-import DataFormatMixin from './mixins/DataFormat';
-import DisplayErrors from './common/DisplayErrors';
-import Editor from './Editor'
-import throttle from 'lodash/throttle';
+import { createUniqIdsMixin } from "vue-uniq-ids";
+import throttle from "lodash/throttle";
+import ValidationMixin from "./mixins/validation";
+import DataFormatMixin from "./mixins/DataFormat";
+import DisplayErrors from "./common/DisplayErrors";
+import Editor from "./Editor";
 
 const uniqIdsMixin = createUniqIdsMixin();
 
 export default {
-  inheritAttrs: false,
   components: {
     DisplayErrors,
     Editor
   },
   mixins: [uniqIdsMixin, ValidationMixin, DataFormatMixin],
-
+  inheritAttrs: false,
   props: [
-    'label',
-    'error',
-    'name',
-    'value',
-    'helper',
-    'controlClass',
-    'richtext',
-    'readonly',
-    'rows'
+    "label",
+    "error",
+    "name",
+    "value",
+    "helper",
+    "controlClass",
+    "richtext",
+    "readonly",
+    "rows"
   ],
+  data() {
+    return {
+      objectOfAttrs: {
+        icon: this.$attrs.icon,
```

Comment:
Where or how this property `icon` is used? I didn't find it in HTML TextArea elements nor Tinymce.

If this does not have effect consider to remove it.

refs:
https://www.tiny.cloud/docs/integrations/vue/#tinymcevuejsintegrationquickstartguide
https://www.w3schools.com/tags/tag_textarea.asp
https://www.w3schools.com/tags/ref_standardattributes.asp